alica_tests/test/alica_tests.test

```diff
@@ -1,14 +0,0 @@
-<launch>
-    <arg name="use_gdb" default="$(optenv GDB false)"/>
```

Comment:
we don't need this file?

Reply:
As far as I can tell this file was no longer used now that the tests are just done with CMake+Gtest skipping catkin